src/test/unit/baro_ms5611_unittest.cc

```diff
@@ -153,6 +153,10 @@ bool busRawWriteRegister(const busDevice_t*, uint8_t, uint8_t) {return true;}
 bool busRawWriteRegisterStart(const busDevice_t*, uint8_t, uint8_t) {return true;}
 void busDeviceRegister(const busDevice_t*) {}
 
+uint16_t spiCalculateDivider() {
+	return 2;
```

Comment:
Eh, should be 4 spaces. :space_invader:

Reply:
Sorry about that, my detab script, applied on commits, was ignoring .cc files. Fixed.